.gitignore

```diff
@@ -6,3 +6,5 @@ tests/clover.xml
 
 # packaged app
 build/artifacts
+
+.envrc
```

Comment:
please remove this from the commit... or what is it needed for?

Reply:
Sure! Its a project based file for setting environment variables https://direnv.net/
I added it to my global ignore file.